homeassistant/components/kef/media_player.py

```diff
@@ -298,11 +298,6 @@ def source_list(self):
         """List of available input sources."""
         return self._sources
 
-    @property
-    def available(self):
```

Comment:
Why can you delete this now? Is available defined by _attr_available?

Reply:
Yes, the default behavior in the base `Entity` class:
https://github.com/home-assistant/core/blob/6f8b032a6ed65d43dc10297271341a3436555961/homeassistant/helpers/entity.py#L412-L415